hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -281,6 +292,19 @@ private static int rollback(JavaSparkContext jsc, String instantTime, String bas
     }
   }
 
+  private static int createSavepoint(JavaSparkContext jsc, String commitTime, String user,
+      String comments, String basePath) throws Exception {
+    HoodieWriteClient client = createHoodieClient(jsc, basePath);
+    try {
+      client.savepoint(commitTime, user, comments);
+      LOG.info(String.format("The commit \"%s\" has been savepointed.", commitTime));
+      return 0;
+    } catch (HoodieSavepointException se) {
+      LOG.info(String.format("Failed: Could not create savepoint \"%s\".", commitTime));
```

Comment:
I think we could change to log.warn, wdyt?

Reply:
> I think we could change to log.warn, wdyt?

Yes, warn is better here.